### PR TITLE
Removed unnessasary dependensies and changed function names

### PR DESCRIPTION
**Pull request by yugal-compro**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,27 +1,17 @@
 const express = require('express');
-var exphbs  = require('express-handlebars');
 const courseRouter = require('./routes/courseRouter');
-
 const cors = require('cors');
 
 const app = express();
```
Comment (SusantoMandal) [inline server.js:6]: unnecessary next lines

File: server.js [hunk 2]
```diff
@@ -1,27 +1,17 @@
...
 
 app.use(cors());
 
-
-
 const port = 7000;
```
Comment (SusantoMandal) [inline server.js:9]: declare variables at the top

File: routes/courseRouter.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 router
   .route('/courses')
   .get(getAllCourses)
-  .post(addCourse);
+  .post(addNewCourse);
 
 router
   .route('/courses/:subjectId')
-  .get(getCourse)
+  .get(getACourse)
```
Comment (SusantoMandal) [inline routes/courseRouter.js:12]: change getACourse to getCourse

File: controllers/courseController.js [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
-    const course = getCourseById(subjectId);
+    const course = getCourse(subjectId);
     (course == null)? res.status(404).json({message : 'Course not found'}) : res.status(200).json(course);
 };
 
-const addCourse = (req,res)=>{
-    const courses = modifiedCoursesArray(req.body);
+const addNewCourse = (req,res)=>{
+    const courses = addCourse(req.body);
     fs.writeFile(`C:/express-assignment/data/courses.json`,JSON.stringify(courses),(err)=>{
```
Comment (SusantoMandal) [inline controllers/courseController.js:17]: write operation should be in the model not in controller

File: controllers/courseController.js [hunk 5]
```diff
@@ -32,7 +31,7 @@ const updateCourse = (req,res)=>{
 
 const deleteCourse = (req,res)=>{
     const subjectId = req.params.subjectId * 1;
-    const courses = modifiedCoursesArrayAfterDeletion(subjectId);
+    const courses = updatedCoursesAfterDeletion(subjectId);
     if(courses==null){
       res.status(404).json('The server is unable to find the course in the database.');
     }else{
```
Comment (SusantoMandal) [inline controllers/courseController.js:37]: write operation should be in the model not in controller

File: server.js (hunk 1 above)
Comment (yugal-compro) [reply server.js:6]: oh

File: server.js (hunk 2 above)
Comment (yugal-compro) [reply server.js:9]: okay

File: routes/courseRouter.js (hunk 3 above)
Comment (yugal-compro) [reply routes/courseRouter.js:12]: okay

File: controllers/courseController.js (hunk 4 above)
Comment (yugal-compro) [reply controllers/courseController.js:17]: okay ok

File: routes/courseRouter.js [hunk 6]
```diff
@@ -1,15 +1,15 @@
 const express = require('express');
 const router = express.Router();
-const {getAllCourses,addCourse,getCourse,updateCourse,deleteCourse} = require('./../controllers/courseController');
+const {getAllCourses,addNewCourse,getACourse,updateCourse,deleteCourse} = require('./../controllers/courseController');
 
 router
   .route('/courses')
   .get(getAllCourses)
-  .post(addCourse);
+  .post(addNewCourse);
```
Comment (SusantoMandal) [inline routes/courseRouter.js:8]: change it to addCourse

File: routes/courseRouter.js (hunk 3 above)
Comment (SusantoMandal) [reply routes/courseRouter.js:12]: change getACourse to getCourse

File: routes/courseRouter.js [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 
 router
   .route('/courses/:subjectId')
-  .get(getCourse)
+  .get(getACourse)
   .put(updateCourse)
   .delete(deleteCourse);
```
Comment (SusantoMandal) [inline routes/courseRouter.js:14]: remove unnecessary next lines

**Pull request merged**